src/app/games/martian-chess/MartianChessTutorial.ts

```diff
@@ -0,0 +1,211 @@
+import { TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { Coord } from 'src/app/jscaip/Coord';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+import { MartianChessMove } from './MartianChessMove';
+import { MartianChessPiece, MartianChessState } from './MartianChessState';
+
+const _: MartianChessPiece = MartianChessPiece.EMPTY;
+const A: MartianChessPiece = MartianChessPiece.PAWN;
+const B: MartianChessPiece = MartianChessPiece.DRONE;
+const C: MartianChessPiece = MartianChessPiece.QUEEN;
+
+export const NOT_A_FIELD_PROMOTION: () => string = () => $localize`This is not a field promotion!`;
+
+export class MartianChessTutorial {
```

Comment:
Voir les commentaires tutos que j'avais mis sur le ticket

Reply:
Va falloir me rafraichir derrière les oreilles monsieur le coiffeur !

Reply:
regarde donc le ticket, tout est dedans !

Reply:
pour reprendre le commentaire en question: 

Voici pour une première vague !

    ça serait cool de visualiser les captures (pas juste via les points)
    faire un déplacement de (+1, +2) résulte en une erreur pourrie
    le nombre sur l'horloge rend bizarre, le stroke semble trop gros
    c'est perturbant que si tu "call the clock" puis foire ton mouvement, tu dois à nouveau call the clock !
    du coup y'a ce mouvement illégal du drône en (+1, +1)
    faut générer l'image
    "You cannot do a move that is the reversed of the last one" -> do -> perform, reversed -> reverse. Et c'est le cas même si le mouvement précédent à effectué une capture ?

Tuto steps:

1/12 The goal of Martian Chess is to have the most points at the end of the game. Each piece is assigned a value. You win this value when capturing a piece. The board is divided in two by a line called the canal. There is one side for each player. You do not own pieces, but you control the ones on your side of the canal. Pieces are colored to help you: they will change color when crossing the canal.

2/12 There is -> There are. + points -> dots + point -> dot !

3/12 They can move one step diagonally? Pourquoi insister sur "first player"
et pas simplement dire "you're playing dark"?

4/12 They cone move two steps orthogonally plutôt ? Aussi -> you're playing Dark

5/12 In one of the 8 directions -> in any direction
without ... -> as long as they do not jump over any piece
you play -> you're playing

6/12 When a piece crosses the canal and lands on another piece, you capture it and gain the value of that captured piece. However, you lose control of your piece as it crossed the canal!
"This move was not a capture." -> You did not capture anything.

7/12 It is sometimes possible to perform what is called a <i>field promotion</i>. If you are out of one type of piece, you can merge two pieces to add their value and get a new piece. For example, if you have no drones, you can merge two pawns into a drone. To merge two pawns, move one pawn on the other.
Such a move is possible for Dark. Do it.

8/12 The other kind of field promotion is to merge a drone and a pawn to
obtain a queen.

9/12 When the game seems to be going nowhere, you can "call the clock"
during your turn. To do so, click on the clock symbol on the right, then
perform your move. Once the clock is called, seven more turn will be played.

You're playing Dark. Call the clock and perform a move.

10/12 If a capture is performed after the clock has been called, the countdown
will restart from 7.

11/12 When seven turns have passed pass after the clock has been called, the
player with the most points win. If both players have the same number of points, it is
a tie.

12/12 When a player send their last piece into the opponent's territory, the
game ends and the player with the most points wins. If both players have the
same number of points, the last player that moved wins!

-> comme on voit pas les scores, c'est difficile à dire que clair va vraiment
gagner !

Reply:
1. contre question: on dit "the goal of" ou "the goal at" ou c'est au cas par cas ?
2. restart *from* seven ?
3. "when **a** player send **their** last piece" ou plutôt "when **players** send their last piece" ?
Le "a" et "their" me perturbe sévèèèère :D

Reply:
2. j'ai mis "'the countdown restart"
3. j'ai entourloupé le truc, c'est inclusif sans sonner comme un bible de gutenberg !

Reply:
2. restarts* !